source/reference/customize-prompt.txt

```diff
@@ -0,0 +1,35 @@
+.. _customize-the-mongosh-prompt:
+
+===========================================
+Customize the :binary:`~bin.mongosh` Prompt
```

Comment:
[Note for review]

This page is just refactoring existing copy. It should not need review other than a review of the updated placement of the page in the ToC.